sdk/test-utils/perf/src/testProxyHttpClient.ts

```diff
@@ -10,6 +10,7 @@ import {
   SendRequest
 } from "@azure/core-rest-pipeline";
 import { RequestOptions } from "http";
+import * as https from "https";
```

Comment:
Do `import { Agent } from "https";` instead and update the type?

Reply:
I'd rather leave as is as it makes clear that the `Agent` type is coming from the `https` module in line

Reply:
The intention is that we don't have to pull in the whole of "https" when all we needed was just the agent. :)

Reply:
Sure, I get that. But I'd argue that the readability benefit trumps that slightly in this case. It's a bit of a coin flip though, I will change it. We can go somewhere in the middle with `import { Agent as HttpsAgent } from "https";` too I suppose.

Reply:
I don't mind, do as you please 🙂

Reply:
Middle ground it is!